### PR TITLE
Commit for code review

### PR DESCRIPTION
**Pull request by nikolayhsu**
Project pushed for code review.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    "prod": "rimraf dist && webpack --bail --progress --profile",
+    "build": "rimraf dist && webpack --progress --profile --watch",
+    "server": "webpack-dev-server --history-api-fallback --inline --progress",
+    "test": "karma start",
+    "test-watch": "karma start --auto-watch --no-single-run",
+    "start": "npm run server"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://nikolay_hsu@bitbucket.org/nikolay_hsu/dsiti-search-function.git"
```
Comment (gvanimurthy) [inline package.json]: change this to current repo

File: package.json [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  },
+  "author": "Queensland Government",
+  "license": "MIT",
+  "bugs": {
+    "url": ""
+  },
+  "homepage": "https://bitbucket.org/nikolay_hsu/dsiti-search-function",
```
Comment (gvanimurthy) [inline package.json]: change this to current home page

File: src/public/index.html [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+<div id="nav-section">
+<div class="box-sizing">
+<!--#include virtual = "/dsiti/qsa/assets/includes/search-nav.html"-->
+</div>
+</div>
+</div>
+</div>
+<!--#include virtual="/assets/includes/global/footer.html"-->
+<!-- endnoindex -->
+<script type="text/javascript" src="//cdn.datatables.net/1.10.13/js/jquery.dataTables.min.js"></script>
```
Comment (gvanimurthy) [inline src/public/index.html]: @raymedia are we allowed to use CDN?

File: src/public/index.html [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+<link rel="stylesheet" href="//cdn.datatables.net/1.10.13/css/jquery.dataTables.min.css" />
```
Comment (gvanimurthy) [inline src/public/index.html]: @raymedia are we allowed to use CDN?

**Comment by ghost** [comment]
How we are testing the application?  I could not find any spec for Unit test and  E2E test.

**Comment by nikolayhsu** [comment]
I am working on unit test at the moment.

**Review by gvanimurthy — APPROVED** [review]

**Pull request merged**